src/api.yaml

```diff
@@ -77,6 +77,58 @@ enum:
       openGL: The OpenGL backend
       openGLES: The OpenGL ES backend
 
+  blend factor:
+    description: >
+      Determines how a blend factor is calculated. This factor will be combined
+      with the original color based on the selected blend operation.
+    ref:
+      name: GPUBlendFactor
+      anchor: '#enumdef-gpublendfactor'
+    values:
+      zero: >
+        Factor is set to zero.<br>
+        `(0, 0, 0, 0)`
+      one: >
+        Factor is set to one. <br>
+        `(1, 1, 1, 1)`
+      src: >
+        Factor is set to the source values. <br>
+        `(R_src, G_src, B_src, A_src)`
+      one minus src: >
+        Factor is set to one minus source value. <br>
+        `(1 - R_src, 1 - G_src, 1 - B_src, 1 - A_src)`
+      src alpha: >
+        Factor is set to the source alpha value. <br>
+        `(A_src, A_src, A_src, A_src)`
+      src alpha saturated: >
+        Factor is set to the source alpha value. <br>
+        `(A_src, A_src, A_src, A_src)`
```

Comment:
This one seemed weird to me as it's the same as `src alpha`. I filed https://github.com/gpuweb/gpuweb/issues/3432 to determine the difference between the two.